README.md

```diff
@@ -131,6 +142,70 @@ The `upgrade` command upgrades an existing helm release. A config file has to be
 
 The `version` command pretty much does what it says on the tin: it gives out the version currently used.
 
+<a name="config"></a>
+## Configuration
+
+A `havener config file` provides an easy solution for configurating and deploying one or multiple Helm Charts. The `config` is saved as a `.yml-file` and is used by the `deploy` and `upgrade` commands. Besides information about the charts, it can overide the values.yml file and can contain further pre- and post-processing steps.
+```yml
+name: mongo deployment
+releases:
+- name: mongodb
+  namespace: mongodb
+  version: (( env VERSION ))
+  location: stable/mongodb
+  overrides:
+    mongodbUsername: (( env USERNAME ))
+    mongodbPassword: (( env PASSWORD ))
+    mongodbDatabase: (( env DATABASE ))
+  before: [echo "$(date) before release"]
+  after: [echo "Installed mongoDB with credentials $USERNAME/$PASSWORD"]
+
+env:
+  VERSION: 1
+  USERNAME: (( shell echo "user" ))
+  PASSWORD: (( secret default root-password password.txt ))
+  DATABASE: admin
+
+before:
+- cmd: /bin/bash
+  args:
+  - -c
+  - |
+    #!/bin/bash
+    echo "$(date) before deployment"
+
+after:
+- echo "$(date) after deployment"
+````
+### sections
+#### releases
```

Comment:
I know it is a lot of work, but I can highly recommend using a Markdown linter like `davidanson.vscode-markdownlint` to help you clean up the file.

Reply:
Thanks for the tip, I have added it :)